playground/nextjs-app-router/app/globals.css

```diff
@@ -59,11 +59,24 @@
   }
 }
 
+/* Customize Theme */
+/*
+@layer base {
+  :root,
+  .default-light,
+  .default-dark,
+  .base,
+  .cyberpunk,
+  .minimal {
+    --ock-bg-primary: red;
+  }
+} */
+
```

Comment:
How to customize your theme or create a new one - you'll need to override the specific `--ock` variable